core/src/main/java/com/netease/arctic/op/OverwriteBaseFiles.java

```diff
@@ -34,24 +34,29 @@
 import org.apache.iceberg.expressions.Expressions;
 import org.apache.iceberg.io.CloseableIterable;
 import org.apache.iceberg.relocated.com.google.common.collect.Lists;
+import org.apache.iceberg.relocated.com.google.common.collect.Maps;
 import org.apache.iceberg.util.StructLikeMap;
 
 import java.io.IOException;
 import java.util.Collections;
 import java.util.HashSet;
 import java.util.List;
+import java.util.Map;
 
 /**
  * Overwrite {@link com.netease.arctic.table.BaseTable} and change max transaction id map
  */
 public class OverwriteBaseFiles extends PartitionTransactionOperation {
 
+  public static final String PROPERTIES_TRANSACTION_ID = "txId";
+
   private final List<DataFile> deleteFiles;
   private final List<DataFile> addFiles;
   private final List<DeleteFile> deleteDeleteFiles;
   private final List<DeleteFile> addDeleteFiles;
   private Expression deleteExpression = Expressions.alwaysFalse();
   private final StructLikeMap<Long> maxTransactionId;
+  private final Map<String, String> summary = Maps.newHashMap();
```

Comment:
I has not found where the field summary use?

Reply:
yes, it is depelicated with ` PartitionTransactionOperation.properties`, I would drop it